Source/Fuse.Controls.Primitives/TextControls/TextControl.Forwarding.uno

```diff
@@ -86,7 +86,13 @@ namespace Fuse.Controls
 		{
 			OnPropertyChanged(FontSizePropertyName);
 			var edit = GetITextView();
-			if (edit != null) edit.FontSize = FontSize;
+			if (edit != null)
+			{
+				if defined(IGNORE_FONT_SCALING)
```

Comment:
Wouldn't it be better/more correct to have this `if defined(IGNORE_FONT_SCALING)` inside `FontSizeScaled`'s `get`'er, in one place, instead of duplicated at every call site?

Reply:
I am sorry, I’ve done that because you said in the last comment review to do:

_Then do a search for all call sites of FontSize and evaluate per call site whether to read FontSizeScaled or FontSize_

So in my understanding about that is I have to find all sites that call FontSize and do evaluate whether we use FontSize or FontSizeScaled based on an On/Off compiler flag

Correct me if I’m wrong, So if we do this, will it change all the call sites from using `FontSize` become`FontSizeScaled`?